GameEngine/GameState.cs

```diff
@@ -24,27 +24,27 @@ public GameState(IPlayer player, Deck deck, int gameSizeMultiplier)
         public void StartGame()
         {
             var hand = Deck.Draw(3);
-            Player.AddCardsToHand(hand);
+            Player.Hand.Add(hand);
```

Comment:
💡 `Hand.Add(hand)` reads a bit funny. What do you think about `Hand.Add(cards)`?